README.md

```diff
@@ -76,3 +83,9 @@ The last bit of work is making sure the templates render nicely.
 This will render the templates in
 
 https://github.com/GetStream/Stream-Example-Py/tree/master/templates/activity
+
+## Copyright and License Information
+
+Copyright (c) 2014-2017 Stream.io Inc, and individual contributors. All rights reserved.
+
+See the file "LICENSE" for information on the history of this software, terms & conditions for usage, and a DISCLAIMER OF ALL WARRANTIES.
```

Comment:
maybe mention "bsd" somewhere

Reply:
Github detects the BSD-3 clause without the explicit title. I'll leave it like this for consistency (for now).

E.g: https://github.com/GetStream/stream-js/blob/master/LICENSE